src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -216,6 +217,12 @@
   </div>
 </template>
 
+<script lang="ts">
+const IS_MAYBE_MAC = typeof window !== 'undefined' && window.navigator.platform.toLowerCase().includes('mac')
```

Comment:
suggestion: a little better useragent check
```suggestion
const IS_MAYBE_MAC = typeof window !== 'undefined' && /macOS|Mac|iPhone|iPod|iPad/i.test(navigator?.userAgentData?.platform)
```